torch/_dynamo/utils.py

```diff
@@ -755,6 +765,16 @@ def dict_const_keys(value):
     return set(k for k in value.keys() if not isinstance(k, torch.nn.Parameter))
 
 
+def dict_const_keys_repr(const_keys):
+    if any(isinstance(k, enum.Enum) for k in const_keys):
+        const_keys_str = f"{set([enum_repr(k) if isinstance(k, enum.Enum) else repr(k) for k in const_keys])}".replace(
```

Comment:
I think it would be good to add a comment about what this is doing, it can be tricky to understand just by looking at it (ie removing quotes to render enum in guard code)